packages/client/src/build.rs

```diff
@@ -364,6 +364,38 @@ impl Outcome {
 			Self::Succeeded(value) => data::Outcome::Succeeded(value.data(tg).await?),
 		})
 	}
+
+	pub fn terminated(&self) -> bool {
+		if let Outcome::Terminated = self {
+			true
+		} else {
+			false
+		}
+	}
+
+	pub fn canceled(&self) -> bool {
```

Comment:
All these methods are added for completeness.

Reply:
Could use `matches!()`?

Reply:
Agreed, these methods are unnecessary. I think to be idiomatic it should be `is_canceled` etc.